src/Symfony/Component/Cache/Adapter/PdoAdapter.php

```diff
@@ -12,6 +12,7 @@
 namespace Symfony\Component\Cache\Adapter;
 
 use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver\PDO\Exception;
```

Comment:
Sorry I wasn't clear enough: we don't want to create a new dependency on dbal. Instead we should take inspiration frombthe code there.

Reply:
My bad. Please take a look again.